modules/Bio/EnsEMBL/Variation/DBSQL/vcf_config.json

```
@@ -858,6 +873,14 @@
       "filename_template" : "https://ftp.ensembl.org/pub/release-106/compara/conservation_scores/91_mammals.gerp_conservation_score/gerp_conservation_scores.felis_catus.Felis_catus_9.0.bw",
       "annotation_type": "gerp"
     },
+    {
+      "id": "27_sauropsids.gerp_conservation_score",
+      "species": "ficedula_albicollis",
+      "assembly": "FicAlb1.5",
+      "type": "remote",
+      "filename_template" : "https://ftp.ensembl.org/pub/release-106/compara/conservation_scores/27_sauropsids.gerp_conservation_score/gerp_conservation_scores.ficedula_albicollis.FicAlb1.5.bw",
```

I think this needs to be updated... Discussing this with Diana, but I will leave this here for now.

```suggestion
      "filename_template" : "https://ftp.ensembl.org/pub/release-108/compara/conservation_scores/27_sauropsids.gerp_conservation_score/gerp_conservation_scores.ficedula_albicollis.FicAlb1.5.bw",
```

---

Make sense!

---

This needs to be updated in a few weeks for release 108.